packages/camera/camera_platform_interface/lib/src/types/image_file_format.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The format in which images should be returned from the camera.
+enum ImageFileFormat {
+  /// This is the default value. It will return the image in JPEG format.
+  /// Is the widely used format for images.
```

Comment:
This should follow: https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph. Same for `heif` below.

Also, I don't think this should mention that this is the default value. You should add a comment to `setImageFileFormat` that JPEG is the default.